build/utils.py

```diff
@@ -203,7 +220,8 @@ def process_accelerate_launch_args(job_config_dict):
         )
 
     # Add training_script
-    accelerate_launch_args.append("/app/launch_training.py")
+    script = os.environ.get("LAUNCH_TRAINING_SCRIPT") or "/app/launch_training.py"
```

Comment:
nit: slightly cleaner way of doing this
```suggestion
    script = os.environ.get("LAUNCH_TRAINING_SCRIPT", "/app/launch_training.py")
```
which does for you that if `LAUNCH_TRAINING_SCRIPT` doesn't exist, set it to `"/app/launch_training.py"`